### PR TITLE
Update DCR and Try Out page

### PR DESCRIPTION
**Pull request by silvashenol20**
Fix the review comments in sms otp page with sample configurations and try out page with valid sample request and responses.

### CODE REVIEW

File: en/docs/get-started/try-out-flow.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ The Accredited Data Recipient application redirects the bank customer to authent
       "typ": "JWT"
     }
     {
-      "aud": "https://localhost:9446/oauth2/token",
-      "iss": "3nzwb1VuaUISzujNe6QjDxlgnCka",
+      "aud": "<This is the audience that the ID token is intended for. Example, https://<IS_HOST>:9446/oauth2/token>",
```
Comment (isharailanga) [inline en/docs/get-started/try-out-flow.md:91]: do the same changes in the try-out/account-and-transaction-flow/ doc.

File: en/docs/get-started/try-out-flow.md [hunk 2]
```diff
@@ -104,11 +104,11 @@ The Accredited Data Recipient application redirects the bank customer to authent
...
        "response_type": "code id_token",
-       "redirect_uri": "https://wso2.com",
+       "redirect_uri": "<CLIENT_APPLICATION_REDIRECT_URI>",
        "state": "suite",
-       "exp": 1739640532,
-       "nonce": "8fc4cbb4-287b-42aa-a1d0-67dce6fc7479",
-       "client_id": "3nzwb1VuaUISzujNe6QjDxlgnCka"
+       "exp": <EPOCH_TIME_OF_TOKEN_EXPIRATION>,
+       "nonce": "<PREVENTS_REPLAY_ATTACKS>",
+       "client_id": "<CLIENT_ID>"
```
Comment (isharailanga) [inline en/docs/get-started/try-out-flow.md:111]: When giving samples for the scope in the `Authorization URL` (not here) I think it was suggested to link to the CDS Scopes page and use a placeholder. 

https://consumerdatastandardsaustralia.github.io/standards/#authorisation-scopes

shall we do that change as well?

**Comment by isharailanga** [comment]
Please check and verify if we need to add the executors shared with us.  Please set up a pack (do not run configure.sh) and see if those executors are there. If not we need to add them to the docs. If they are there, we can avoid documenting that.

File: en/docs/get-started/try-out-flow.md (hunk 1 above)
Comment (silvashenol20) [reply en/docs/get-started/try-out-flow.md:91]: This PR is to fix the above review comment [https://github.com/wso2/docs-ob-toolkit-cds/pull/16](https://github.com/wso2/docs-ob-toolkit-cds/pull/16)

**Review by isharailanga — APPROVED** [review]

File: en/docs/get-started/try-out-flow.md (hunk 1 above)
Comment (isharailanga) [reply en/docs/get-started/try-out-flow.md:91]: Merging this PR as other changes will be addressed in https://github.com/wso2/docs-ob-toolkit-cds/pull/16

File: en/docs/get-started/try-out-flow.md (hunk 2 above)
Comment (isharailanga) [reply en/docs/get-started/try-out-flow.md:111]: Is fixed in https://github.com/wso2/docs-ob-toolkit-cds/pull/16

**Pull request merged**